src/app/inventory/advanced-write-actions.ts

```diff
@@ -299,7 +299,11 @@ async function getAwaToken(
       }
       */
     } catch (e) {
-      throw new DimError('AWA.FailedToken').withError(e);
+      const dimError = new DimError('AWA.FailedToken');
+      if (e instanceof Error) {
+        dimError.withError(e);
+      }
+      throw dimError;
```

Comment:
Why not `.withError(convertToError(e))` here?

Reply:
Because I'd already switched those all before I came up with `convertToError` :-)